docs/k8s-helm.md

```diff
@@ -125,61 +193,70 @@ helm -n $NS upgrade -f values.yaml $RELEASE timeplus/timeplus-enterprise
 Please check the [release notes](/enterprise-releases) to confirm the target version of Timeplus Enterprise can be upgraded in-place, by reusing the current data and configuration. For example [2.3](/enterprise-v2.3) and [2.4](/enterprise-releases) are incompatible and you have to use migration tools.
 
 If you confirm you can upgrade to the new version, you can run the following commands to upgrade to the latest version:
+
 ```bash
 helm repo update
 helm -n $NS upgrade $RELEASE timeplus/timeplus-enterprise
 ```
 
 You can also check the available versions and upgrade to a specific version:
+
 ```bash
 helm repo update
 helm search repo timeplus -l
 helm -n $NS upgrade $RELEASE timeplus/timeplus-enterprise --version va.b.c
 ```
 
-### Uninstall and cleanup
-
-To uninstall the helm release, just run `helm -n $NS uninstall $RELEASE` to uninstall it.
+### Prometheus metrics
```

Comment:
I noticed you removed a few sections. Probably need add them back
- Uninstall and cleanup
- User management

Reply:
`Uninstall and cleanup` is moved above. For User management, let's use https://github.com/timeplus-io/docs/pull/264#discussion_r1898748254 to discuss